cep-9999.md

```diff
@@ -0,0 +1,165 @@
+<table>
+<tr><td> Title </td><td> Serving run_exports metadata in conda channels </td>
+<tr><td> Status </td><td> Draft </td></tr>
+<tr><td> Author(s) </td>
+<td> 
+    Jaime Rodríguez-Guerra &lt;jrodriguez@quansight.com&gt;
+</td></tr>
+<tr><td> Created </td><td> May 4, 2023</td></tr>
+<tr><td> Updated </td><td> May 4, 2023</td></tr>
+<tr><td> Discussion </td><td> https://github.com/conda-incubator/ceps/pull/51 </td></tr>
+<tr><td> Implementation </td><td> NA </td></tr>
+</table>
+
+## Abstract
+
+Have conda channels serve standalone `run_exports` metadata, next to `repodata.json`.
+
+## Motivation
+
+Building infrastructure (such as some conda-forge bots) use `run_exports` to calculate which packages need rebuilding as part of an upgrade. Right now, conda-forge needs to maintain their own JSON database, which involves downloading and extracting the new artifacts as they become available: the `run_exports` metadata lives inside the `.tar.bz2` and `.conda` files.
+
+Since `conda-index` already processes the package metadata to generate `repodata.json`, it would be trivial to also generate `run_exports.json` and serve them together.
+
+## Precedence and role in the pinning resolution process
+
+This file is not meant to replace the `run_exports` metadata already present in the package archives. It merely presents that information in a more convenient way. 
+`conda-build`-like clients are free to use either the `run_exports` metadata in the archives or the one in `run_exports.json`, since they should be equivalent.
+Special keys like `pin_run_as_build` will keep their behavior, since `run_exports.json` does not add a new level of precedence in the pinning resolution process. Again, it's an equivalent source for information already present in the archives.
+
+This means that `run_exports.json` should not be patched (like it is done with `repodata.json`);
```

Comment:
I'm wondering if we should adopt [IETF BCP 14 convention](https://datatracker.ietf.org/doc/html/rfc8174#section-2) and use an explicitly capitalized "SHOULD NOT" here. (Or possibly a stronger "MUST NOT".)